code/modules/research/fabricators.dm

```diff
@@ -297,6 +297,7 @@
 			*/
 			being_built.forceMove(L) //Put the thing in the lockbox
 			L.name += " ([being_built.name])"
+			L.fits_max_w_class = 0 //You can't put things back into these boxes to prevent breaking arbitrary items
```

Comment:
But why would putting the laser in a lockbox break it? After you've broken it, the lockbox can't break again.